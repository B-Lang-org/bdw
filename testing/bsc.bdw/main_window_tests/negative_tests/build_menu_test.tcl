
# Opens the specified project
handlers::open_project project.bspec

# Compiles the project
handlers::compile 

# Sets the compilation type
handlers::set_compilation_type bsc

# Compiles the project
handlers::compile 

# Sets the compilation type
handlers::set_compilation_type make 

# Compiles the current project
handlers::compile

# Links the project
handlers::link 

# Simulates the project
handlers::simulate

# Sets the compilation type
handlers::set_link_type make

# Compiles the project
handlers::link 

# Simulates the project
handlers::simulate

# Sets the compilation type
handlers::set_link_type custom_command

# Compiles the project
handlers::link 

# Simulates the project
handlers::simulate

# Closes the project
handlers::close_project

# Exits the workstation
exit

