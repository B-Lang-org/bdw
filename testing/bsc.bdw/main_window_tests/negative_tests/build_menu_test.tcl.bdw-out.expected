The project top file is not specified.
The project top file is not specified.
The project Makefile is not specified.
The current project has not been compiled yet.
 Please compile!
The simulation executable does not exist
Makefile for the link command is not specified.
The simulation target is not specified
The custom command for linking is not specified.
The custom command for simulation  is not specified
